index.html

```diff
@@ -3865,10 +3863,9 @@ <h3>Linked Data Proofs</h3>
 easily merged into the existing graph of knowledge. Linked Data is
 extensible in a decentralized way, greatly reducing barriers to large scale
 integration. The data model in this specification works well with the
-<a href="https://w3c-dvcg.github.io/ld-proofs/">Linked Data Proofs</a>,
-<a href="https://w3c-dvcg.github.io/ld-signatures/">Linked Data Signatures</a>,
+<a href="https://w3c-ccg.github.io/ld-proofs/">Linked Data Proofs</a>
```

Comment:
I wonder if there is a reason these links are not like the others, e.g.,
```suggestion
Linked Data Proofs [[?LD-PROOFS]]
```